src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -138,15 +138,15 @@ export const DirectDepositContent = ({
 
   // When direct deposit is not set up, we will show
   const notSetUpContent = (
-    <>
-      <button
-        onClick={() => {
-          toggleEditState();
-        }}
-      >
-        Set up direct deposit
-      </button>
-    </>
+    <button
+      className="va-button-link"
+      ref={editBankInfoButton}
```

Comment:
without adding this ref the app would crash when returning from the edit view back to the not-set-up state